modules/browser/BrowserCapabilities.js

```diff
@@ -230,7 +234,10 @@ export default class BrowserCapabilities extends BrowserDetection {
      * https://bugs.webkit.org/show_bug.cgi?id=231071 and https://bugs.webkit.org/show_bug.cgi?id=231074 for details.
      */
     supportsVP9() {
-        return this.isChromiumBased() || this.isReactNative();
+        // #bloomberg #deviceSupport @wliang67 support vp9 for all browsers.
+        // return this.isChromiumBased() || this.isReactNative();
+        return true;
```

Comment:
If you read through the linked tickets, you'll notice that Firefox still has bugs in its VP9 support. We can probably say -

```
return !this.isFirefox();
```

Reply:
Are you referring to https://bugzilla.mozilla.org/show_bug.cgi?id=1492500 ? That seems to indicate firefox should be fine. Is there something else you found?

Reply:
One of the comments on that bug report - https://bugzilla.mozilla.org/show_bug.cgi?id=1492500#c14 mentions https://bugzilla.mozilla.org/show_bug.cgi?id=1633876 that also looks like something Firefox needs to support before VP9 can be enabled.

Reply:
Missing the simulcast is pretty egregious.  Yeah lets do `!firefox` and add the second bugzilla link to the comment.